ui/cypress/support/commands.ts

```diff
@@ -307,40 +508,67 @@ function wrapConnections() {
 }
 
 function wrapDashboards() {
-  return cy
-    .request('GET', '/chronograf/v1/dashboards')
-    .then(({body: response}) => {
-      cy.wrap(response.dashboards).as('dashboards')
-    })
+  return cy.request('GET', `${apiUrl}/dashboards`).then(({body: response}) => {
+    cy.wrap(response.dashboards).as('dashboards')
+  })
 }
 
 function wrapOrgs() {
   return cy
-    .request('GET', '/chronograf/v1/organizations')
+    .request('GET', `${apiUrl}/organizations`)
     .then(({body: response}) => {
       cy.wrap(response.organizations).as('orgs')
     })
 }
 
 function wrapInfluxDBUsers(sourceId: string) {
   return cy
-    .request('GET', `/chronograf/v1/sources/${sourceId}/users`)
+    .request('GET', `${apiUrl}/sources/${sourceId}/users`)
     .then(({body: response}) => {
       cy.wrap(response.users).as('influxDBUsers')
     })
 }
 
 function wrapInfluxDBRoles(sourceId: string) {
   return cy
-    .request('GET', `/chronograf/v1/sources/${sourceId}/roles`)
+    .request('GET', `${apiUrl}/sources/${sourceId}/roles`)
     .then(({body: response}) => {
       cy.wrap(response.roles).as('influxDBRoles')
     })
 }
 
+function wrapChronografUsers() {
+  return cy.request('GET', `${apiUrl}/users`).then(({body: response}) => {
+    cy.wrap(response.users).as('chronografUsers')
+  })
+}
+
+/**
+ * Set application to a default state.
+ */
+export function flush() {
```

Comment:
`Flush` usually means to save unstored data, please rename this function to match its semantics, for example: `reset` or `toInitialState` are good candidates.

Reply:
Function renamed to `toInitialState`.